src/JurorsRegistry.ts

```diff
@@ -46,16 +46,19 @@ export function handleJurorActivated(event: JurorActivated): void {
 export function handleJurorDeactivationRequested(event: JurorDeactivationRequested): void {
   updateJuror(event.params.juror, event)
   createANJMovementForTerm(event.params.juror, DEACTIVATION, event.params.amount, event.params.availableTermId, event)
+  increaseTotalDeactivation(event.address, event.params.amount)
 }
 
 export function handleJurorDeactivationUpdated(event: JurorDeactivationUpdated): void {
   updateJuror(event.params.juror, event)
   createANJMovementForTerm(event.params.juror, DEACTIVATION, event.params.amount, event.params.availableTermId, event)
+  // TODO: update deactivation totalizator
```

Comment:
Why is this a TODO? Shouldn’t it be `decreaseTotalDeactivation(event.address, event.params.amount)`?

Reply:
It was a bit more specific than that, fixed now :)

Reply:
Changes make sense, although so far it seems this event is always for reducing the amount (at least I only found it [here](https://github.com/aragon/aragon-court/blob/master/contracts/registry/JurorsRegistry.sol#L639)). Anyway, yes, also previous amount was needed, too bad that the event wasn’t giving it ;-)